packages/kmod-5.10-nvidia/kmod-5.10-nvidia.spec

```diff
@@ -0,0 +1,200 @@
+
+install kernel/nvidia-drm.mod.o %{buildroot}/%{_cross_datadir}/nvidia/tesla/%{nvidia_tesla_470_version}/module-objects.d
+install kernel/nvidia-drm.o %{buildroot}/%{_cross_datadir}/nvidia/tesla/%{nvidia_tesla_470_version}/module-objects.d
+
+install -m 755 nvidia-smi %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin/%{nvidia_tesla_470_version}
+install -m 755 nvidia-debugdump %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin/%{nvidia_tesla_470_version}
+install -m 755 nvidia-cuda-mps-control %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin/%{nvidia_tesla_470_version}
+install -m 755 nvidia-cuda-mps-server %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin/%{nvidia_tesla_470_version}
+install -m 755 nvidia-ngx-updater %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin/%{nvidia_tesla_470_version}
+
+# TODO: add remaining libraries once we implement "image per variant" in the rpm2img script
+# misc
+# Add libnvidia-ml.so for testing purposes
+install  -m755 libnvidia-ml.so.%{nvidia_tesla_470_version} %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}
+
+ln -s libnvidia-ml.so.%{nvidia_tesla_470_version} %{buildroot}%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}/libnvidia-ml.so.1
+
+popd
+
+%files
+%{_cross_attribution_file}
+%dir %{_cross_libexecdir}/nvidia
+%dir %{_cross_libdir}/nvidia
+%dir %{_cross_datadir}/nvidia
+%dir %{_cross_libdir}/modules-load.d
+%dir %{_cross_factorydir}%{_cross_sysconfdir}/drivers
+%{_cross_tmpfilesdir}/nvidia.conf
+%{_cross_libdir}/systemd/system/
+%{_cross_libdir}/modules-load.d/nvidia-dependencies.conf
+
+%files tesla-470
+%license NVIDIA-Linux-%{_cross_arch}-%{nvidia_tesla_470_version}/LICENSE
+%dir %{_cross_datadir}/nvidia/tesla/%{nvidia_tesla_470_version}
+%dir %{_cross_libexecdir}/nvidia/tesla/bin/%{nvidia_tesla_470_version}
+%dir %{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}
+%dir %{_cross_datadir}/nvidia/tesla/%{nvidia_tesla_470_version}/module-objects.d
+%dir %{_cross_factorydir}/nvidia/tesla/%{nvidia_tesla_470_version}
+
+# Binaries
+%{_cross_libexecdir}/nvidia/tesla/bin/%{nvidia_tesla_470_version}/nvidia-debugdump
+%{_cross_libexecdir}/nvidia/tesla/bin/%{nvidia_tesla_470_version}/nvidia-smi
+
+# Libraries
+%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}/libnvidia-ml.so.1
+%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}/libnvidia-ml.so.%{nvidia_tesla_470_version}
+
+# Configuration files
+%{_cross_factorydir}%{_cross_sysconfdir}/drivers/nvidia-tesla-%{nvidia_tesla_470_version}.toml
+%{_cross_factorydir}/nvidia/tesla/%{nvidia_tesla_470_version}/ld.so.conf
+%{_cross_factorydir}/nvidia/tesla/%{nvidia_tesla_470_version}/nvidia-path.env
+
+# Kernel module objects
+%{_cross_datadir}/nvidia/tesla/%{nvidia_tesla_470_version}/module-objects.d/nvidia.mod.o
+%{_cross_datadir}/nvidia/tesla/%{nvidia_tesla_470_version}/module-objects.d/nv-interface.o
+%{_cross_datadir}/nvidia/tesla/%{nvidia_tesla_470_version}/module-objects.d/nv-kernel.o
```

Comment:
nit: I'd repeat the proposed comment "headers" from the `%install` section for each group of related files